src/Symfony/Component/HttpFoundation/Tests/Session/Storage/NativeSessionStorageTest.php

```diff
@@ -207,19 +207,39 @@ public function testCookieOptions()
         $this->assertEquals($options, $gco);
     }
 
-    public function testSessionOptions()
+    public function testCacheExpireOption()
     {
         $options = [
-            'trans_sid_tags' => 'a=href',
             'cache_expire' => '200',
         ];
 
         $this->getStorage($options);
 
-        $this->assertSame('a=href', \ini_get('session.trans_sid_tags'));
         $this->assertSame('200', \ini_get('session.cache_expire'));
     }
 
+    /**
+     * The test must only be removed when the "session.trans_sid_tags" option is removed from PHP or when the "trans_sid_tags" option is no longer supported by the native session storage.
+     */
+    public function testTransSidTagsOption()
+    {
+        set_error_handler(function ($errno, $errstr) {
+            if ('ini_set(): Usage of session.trans_sid_tags INI setting is deprecated' !== $errstr) {
+                throw new \Exception($errstr);
```

Comment:
Should we rather call the previous error handler instead?

Reply:
I have pushed something. You mean like this?